.github/workflows/publish-main.yaml

```diff
@@ -0,0 +1,75 @@
+name: Publish to main 
+
+on:
+  push:
+    branches:
+      - 'Branch Name'
+  
+env:
+  GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }} 
+  release-version: "0.0.0"
+  community_operators_dir_path: "../community-operators/operators/gingersnap/"
+  community_operators_prod_dir_path: "../community-operators-prod/operators/gingersnap/"
+
+  
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Configure Git
+        run: |
+          git config --global user.name "pminz"
+          git config --global user.email "pminz@redhat.com"
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with: 
+           path: operator
+
+
+      - name: Checkout community-operators repo
+        uses: actions/checkout@v2
+        with:
+          repository: k8s-operatorhub/community-operators
+          path: community-operators
+          ref: main
+          token: ${{ secrets.PAT_TOKEN }}
+
+      - name: Check if new version directory exists in community-operators repo
+        id: check_dir_1
+        run:  |
+          cd operator 
+          ./.github/scripts/directory.sh ${{ env.community_operators_dir_path}} ${{ env.release-version}}
+
+      - name: Checkout community-operators-prod repo
```

Comment:
There's no need for us to check `community-operators-prod` repo as we don't rely on this in the upstream CI.